### ISSUE TITLE
TrailingCommaInArguments force comma only for hash as last argument

### ISSUE THREAD

**Issue by printercu** [rubocop-hq/rubocop#5252]
Hi!

`Style/TrailingCommaInArguments` cop with `EnforcedStyleForMultiline` set to `comma` or `consistent_comma` warns about missing comma for single argument, when it's had to be placed to separate line:

```ruby
where(
  arel_table[:name].matches(likestr).or(arel_table[:email].matches(likestr))
)

Some::Very::Long::External::Constant.and_long_method(
  single_short_arg
)
```

It seems like it maybe unwanted to force coma for every such method call: this may be required only if method accepts splat arguments, but will be confusing when method accepts fixed number of arguments and had to be typed on multiple lines due to length of variables or something else.

Having this, i think the only case when it's desired to have a trailing coma, is a parenthesized method call where the last argument is a hash. I've not found any options to configure rubocop this way.

```ruby 
# bad
method(arg,
  opt: 1
)

# good

# accepts fixed number of args (ignore)
method(
  arg
)

# accepts splat args (ignore)
method(
  arg,
)

# last arg is hash - force trailing comma
method(arg,
  opt: 1,
)
```

WDYT?

Thank you!

**Comment by printercu** [comment]
I've found bunch of issues related to TrailingCommaInArguments (#3392 #2847 #2846). I hope the proposed way will solve them.

**Comment by bbatsov** [comment]
There hasn't been much activity on this ticket and our Core Team is spread too thin on so many tasks, so I'll just close it for the sake of having a cleaner lists of tasks to focus on. We'd gladly review a PR, but it's unlikely that'd we're going to tackle this ourselves in the foreseeable future.
